python/pyspark/mllib/tree.py

```diff
@@ -305,31 +330,41 @@ def trainClassifier(cls, data, numClasses, categoricalFeaturesInfo, numTrees,
         Method to train a decision tree model for binary or multiclass
```

Comment:
`decision tree model` should be `random forest model` (applies also to random forest regression below).

I actually prefer the format in `DecisionTree` above: `Train a random forest model for classification.`, i.e. dropping the `Method to` part.